solarkraft/src/instrument.ts

```diff
@@ -110,6 +75,28 @@ export function instrumentMonitor(
     return { ...monitor, modules: [extendedModule] }
 }
 
+// Decode a Native JS value `v` into its corresponding Apalache IR type.
+function tlaJsonTypeOfNative(v: any) {
+    switch (typeof v) {
+        case 'string':
+            return 'TlaStr'
+        case 'number':
+            return 'TlaInt'
+        case 'bigint':
+            return 'TlaInt'
+        case 'boolean':
+            return 'TlaBool'
+        // an array or object
+        // TODO(#46): support composite types (sequence, tuple, record, set, map)
+        // if (Array.isArray(o)) ...
+        case 'object':
+        case 'symbol':
```

Comment:
symbols are converted to strings in the native conversion of Stellar SDK

Reply:
[6d8f8eb](https://github.com/freespek/solarkraft/pull/65/commits/6d8f8eb7243a3e3d6f0d67899836b5bc33074c71)